### PR TITLE
Make smooth menu animation + fix bg bug

### PR DESCRIPTION
**Pull request by smyzin**
- made a mobile menu absolute
- added smooth animation transition to it
- fix bg for FF

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ module.exports = {
         'mobile-card': '92%',
       },
       height: {
+        280: '17.5rem',
```
Comment (T-Damer) [inline tailwind.config.js:45]: Can we use `h-70`?
If `h-4` is `1rem`, then `17.5 * 4 = 70`rem

**Review by T-Damer — APPROVED** [review]
lgtm, only small fixes needed

File: tailwind.config.js [hunk 2]
```diff
@@ -173,6 +174,8 @@ module.exports = {
         '-96': '-24rem',
         '-1/5': '-20%',
         '1/5': '20%',
+        58.5: '3.661rem',
```
[the remark this answers is not in the document]
Comment (smyzin) [reply tailwind.config.js:178]: `3.66` will be enough. Thanks

File: tailwind.config.js (hunk 1 above)
Comment (smyzin) [reply tailwind.config.js:45]: Renamed it. Thx :)

**Review by T-Damer — APPROVED** [review]
lgtm

**Pull request merged**